pkgs/by-name/xe/xephem/package.nix

```
@@ -0,0 +1,94 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchpatch,
+  makeDesktopItem,
+  copyDesktopItems,
+  installShellFiles,
+  motif,
+  openssl,
+  groff,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xephen";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "XEphem";
+    repo = "XEphem";
+    rev = version;
+    hash = "sha256-TuzXrWoJOAHg31DrJObPcHBXgtqR/KWKFRsqddPzL4c=";
+  };
+
+  nativeBuildInputs = [
+    motif
+    openssl
+    groff
+    xorg.libXmu
+    xorg.libXext
+    xorg.libXt
+    copyDesktopItems
+    installShellFiles
+  ];
```

```suggestion
  nativeBuildInputs = [
    copyDesktopItems
    installShellFiles
    groff # nroff
  ];

  buildInputs = [
    motif
    openssl
    xorg.libXmu
    xorg.libXext
    xorg.libXt
  ];
```
Runtime dependencies go in buildInputs.